errgroup/go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/fullstorydev/go/errgroup
 
-go 1.18
+go 1.20
```

Comment:
Should we do the same for `eventstream`, now that we're not testing it with Go 1.18 in CI anymore?

Reply:
Eh, I think it's fine.  Having no downstream deps covers a multitude of sins.  If someone ever reports a bug that it doesn't work on an old version we can bump.